CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
 # Contributing Guidelines
-* You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
+* You can join our [ AnitaB.org Open Source Zulip Channel](https://anitab-org.zulipchat.com/).  Each active repo has its own channel to direct questions to (for example [#powerup](https://anitab-org.zulipchat.com/#narrow/stream/222537-powerup) or [#portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal)).  
 * Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](http://www.systers.io/code-of-conduct.html).
 ## General Guidelines
-* If you’re just getting started work on an issue labeled “First Timers Only” in any project. Additional resources are available on our [website](http://www.systers.io).
```

Comment:
systers.io is still active http://systers.io/
anitab.org does not have any information about open source. We will eventually build a open source website to replace systers.io, but that is future

Reply:
I think removing it for now until we build out the open source one is better.

Reply:
@isabelcosta  So, should I remove this link, as May suggested. Or let it link to systers.io ?

Reply:
do remove it @Harirai . In this case @mayburgos knows best ;)

Reply:
@isabelcosta  Okay I'll remove it.